### ISSUE TITLE
hotgraphic: fails a11y validation due to having no no child elements with role="listitem"

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2426]
### Subject of the issue/enhancement/features
Hot graphic in 'normal' mode (i.e. `_useGraphicsAsPins: false`) fails a11y validation with the following error:
>Elements with [role] that require specific children [role]s, are missing. 

This is because `.hotgraphic-graphic` has `role="list"`, but there are no child elements with `role="listitem"` which are required
